openlibrary/templates/site/footer.html

```diff
@@ -34,12 +34,12 @@
 
     <!-- clear the window.q and setup the jQuery plugins -->
     <script>
-    \$( function () {
-        var i;
+    window.q.push( function () {
+    var i;
         for ( i = 0; i < window.q.length; i++ ) {
-            \$( window.q[i] );
+            ( window.q[i] );
```

Comment:
I think we may want to keep `\$` before window.q[i], no?

Reply:
Not necessary it still works

Reply:
Even though this is run at the end of the page, there is still no 100% guarantee that the DOM is ready. For instance if HTML is added below the footer, this would not be ready for enhancing with jquery. Better to keep this per @mekarpeles advice.

Reply:
Done